config/routes.rb

```diff
@@ -4,7 +4,13 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root 'users#index'
-  resources :users, only: [:index,:show] do
-    resources :posts, only: [:index,:show]
-end
+  get '/users', to: 'users#index', as: :users
+  get '/users:id', to: 'users#show', as: :user
+  get 'users:user_id/posts', to: 'posts#index', as: :user_posts
+  get 'users:user_id/posts:id', to: 'posts#show', as: :user_post
+  get '/posts/new', to: 'posts#new', as: :new_user_post
+  post '/posts/new', to: 'posts#create' #create_new_post
+  post 'comments/new', to: 'comments#create', as: :new_user_comment #create_new_comment
+  post 'likes/new', to: 'likes#create', as: :new_user_like #create_new_like
+  get 'users/:user_id/post/:id' => 'comments#new', as: 'user_comment'
 end
```

Comment:
[OPTIONAL] I noticed that in your `/config/database.yml` file, you have duplicateed code for the `test` configuration.

![image](https://user-images.githubusercontent.com/38088818/232194811-0746d975-0c61-494c-93f1-10779cc2eabb.png)

- I highly suggest that you move all the code between `line 90 - 95` to `line 61` to avoid having duplicated configuration.